src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -354,12 +371,21 @@ export default class rd2EntryForm extends LightningElement {
     isCurrencySupported() {
         let currencyCode;
 
-        if (this.isMultiCurrencyEnabled) {
-            currencyCode = this.template.querySelector('lightning-input-field[data-id="currencyField"]').value;
+        try {
+            if (this.isMultiCurrencyEnabled) {
+                currencyCode = this.template.querySelector('lightning-input-field[data-id="currencyField"]').value;
+
+                // Sometimes when Edit is clicked in a sequence, the "currencyField" field is not visible,
```

Comment:
@andrewyu-salesforce , @npsp-reedestockton : when Edit is clicked then Cancel and Edit again, then the currency code field on the form is not available , not sure why, thus getting the value from the wired RD record.
@rponti-sforg , when testing edit WIs, please ensure test charters have the multi-currency org testing.